packages/plugin-ext/src/main/browser/view/tree-view-widget.tsx

```diff
@@ -428,6 +428,18 @@ export class PluginTreeModel extends TreeModelImpl {
 
 @injectable()
 export class TreeViewWidget extends TreeViewWelcomeWidget {
+    async refresh(items: string[] | undefined): Promise<void> {
```

Comment:
Minor: It might make more sense to use `?` here, as I would much rather call `tree.refresh()` than `tree.refresh(undefined)`.
```suggestion
    async refresh(items?: string[]): Promise<void> {
```